notebooks/spec2vec/tutorial.ipynb

```diff
@@ -33,7 +33,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
```

Comment:
You used this notebook to test the code right?
the token is still visible here

Reply:
Yes, i used it to visualize the output.

I deleted all the lines I added. What do you mean by the token is visible?